### PR TITLE
hadening/70_fix_administration_and_configuration_manual

### PR DESCRIPTION
**Pull request by frbattid**
* Fixes issues related to #70 
* No unit tests nor e2e tests are required
* **NO CHANGES_NEXT_RELEASE UPDATE SINCE THIS IS A "TRY AND ERROR" EDITION LOOP**
* Assignee @fgalan

### CODE REVIEW

File: doc/manuals/administration_and_configuration/batch/annex_a_create_and_install_keypair.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ You will be prompted for a file path containing the private key and a passphrase
 
 In order to install this key pair for remote authenticated accesses into a certain host, just copy the public key in the ssh authorized keys file owned by the user we want to be authenticated in the host:
 
-    $ cat `<key_name>`.pub >> /home/`<user>`/.ssh/authorized_keys
+    $ cat <key_name>.pub >> /home/<user>/.ssh/authorized_keys
```
Comment (fgalan) [inline doc/manuals/administration_and_configuration/batch/annex_a_create_and_install_keypair.md]: Alternatively, you can try without the 4-whitespaces indenation and use ```` block instead. We experienced some problems the 4-whitepasces technique at Markdown rendering time (I can elaborate at skype if you want).

Comment (frbattid) [inline doc/manuals/administration_and_configuration/batch/annex_a_create_and_install_keypair.md]: I will have it into account... for another iteration of the process. I mean, my first goal is to see something in readthedocs.org :)

Comment (fgalan) [inline doc/manuals/administration_and_configuration/batch/annex_a_create_and_install_keypair.md]: Probably this change makes sense to be donde in the overall documentation, in a single shoot. Maybe you could create a tech debt issue about it (up to you).

NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**